raiden-dapp/package.json

```diff
@@ -33,7 +33,8 @@
     "vue-router": "^3.1.6",
     "vue-virtual-scroller": "^1.0.10",
     "vuetify": "^2.2.27",
-    "vuex": "^3.3.0"
```

Comment:
I couldn't write on the `package-lock.json` file above on GH's UI as it's too big, but please, go over it, search for `raiden-ts`, and remove all keys but `version` from it (including comma). This can be done more easily using editor's folds or vim's `%`